src/Amazon.CloudWatch.EMF/Model/StorageResolution.cs

```diff
@@ -0,0 +1,16 @@
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Amazon.CloudWatch.EMF.Model
+{
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum StorageResolution
+    {
+        [EnumMember(Value = "60")]
+        STANDARD,
+
+        [EnumMember(Value = "1")]
```

Comment:
Why are we having string values here? 
This will result in {... `"StorageResolution":"1"` ...} but we want {... `"StorageResolution":1 `...}.

Reply:
Changed the value assignment directly to the enum itself